File: C/lt_01_estseq/lt01_estseq_28.c
#include <stdio.h>

int main() {

  double preco, media;

  printf("Insira o valor do Preço: ");
  scanf("%lf", &preco);

  printf("Insira a Media de vendas: ");
  scanf("%lf", &media);

  if ((media >= 1000) && (preco >= 80)) {
    preco *= 0.95;
  } else if ((media < 500) && (preco < 30)) {
    preco *= 1.1;
  } else if ((1000 > media) && (media >= 500) && (80 > preco) &&
             (preco >= 30)) {
    preco *= 1.15;
  }

  printf("O novo preço e %.2lf.\n", preco);
  return 0;
}

File: Python/lt_01_estseq/lt01_estseq_28.py
preco_atual=(float(input("Insira o Preço Atual: ")))
media_mensal=(float(input("Insira a Media Mensal de Vendas: ")))

if media_mensal>=1000 and preco_atual>=80:
    preco_atual*=0.95
elif media_mensal<500 and preco_atual<30:
    preco_atual*=1.1
elif 1000>media_mensal>=500 and 80>preco_atual>=30:
    preco_atual*=1.15

print("O novo preço é",preco_atual,"!")
